common/config/presets/c_base_node_mining_deny_methods.toml

```diff
@@ -0,0 +1,37 @@
+# Uncomment all gRPC server methods that should be denied default (only active when `grpc_enabled = true`)
```

Comment:
Please add `,` at the end of each line; this will stop user issues who forget to add the comma